cluster_utils/settings.py

```diff
@@ -181,6 +200,11 @@ def announce_fraction_finished(fraction_finished):
 
 
 def announce_early_results(metrics):
+    """Report intermediate results to cluster_utils.
+
```

Comment:
Potentially add:

"This is used by hyperparameter optimization to stop bad jobs early (`kill_bad_jobs_early` option)."